src/app/state/effects/todos.effects.ts

```diff
@@ -46,9 +40,16 @@ export class TodosEffects {
             console.log(todo)
             return this.todoService.addTodo(todo).pipe(
             tap((item)=>console.log(item)),
-            map((item)=>addTodoSuccess({addedTodo:item.data})
-                ),
-                catchError((err)=>[addTodoFail({error:err})])
+            map((item)=>{
+                this.toastr.success("Add todo successfully","Add todo");
+                return addTodoSuccess({addedTodo:item.data});
```

Comment:
would it be better to use toast inside addTodoSuccess reducer?

Reply:
I add some effects to fix this problem